website/source/docs/job-specification/template.html.md

```diff
@@ -121,5 +121,13 @@ template {
 }
 ```
 
+### Client Configuration
+
+The `template` block has the following [client configuration
+options](/docs/agent/config.html#options):
+
+* `template.allow_host_source` - Allows templates to specify their source
```

Comment:
Making the conscious decision to make Nomad easier to use and keeping the advanced tunables there for the people who need it. The majority of people are running a trusted Nomad cluster